### ISSUE TITLE
Add !help command

### ISSUE THREAD

**Issue by dannycolin** [bnjbvr/botzilla#28]
With Firebot we had a command that let us reply a canned response. For example:

```
!help @username keywords
```

This would be really helpful for when people ask question on #general:mozilla.org on how they can contribute or even on #fx-desktop-community when we want to quickly link to a KB article.

**Comment by bnjbvr** [comment]
Thanks for opening an issue! It would make sense to do so, especially since botzilla has a key/value data store that could map the questions to answers. The `!help` command is already taken by the help module, but that's a detail.

**Comment by dannycolin** [comment]
> The `!help` command is already taken by the help module, but that's a detail.

True! I wonder if it's a good idea in the long run to use generic word like !help as a command since a room could have multiple bots. For example, the github bot use `!gh [command] [arguments]` to avoid collision with other bots. But this is a separate issue on itself :).
